lib/active_record_extended/query_methods/unionize.rb

```diff
@@ -141,7 +144,7 @@ def to_nice_union_sql(color = true)
 
       protected
 
-      def build_unions(arel = @klass.arel_table)
+      def build_unions(arel = @table)
```

Comment:
this method is only called once in the entire gem, i ve tried this default (basically calling `build_unions` without any arguments) and unfortunately it didnt work.

So maybe just remove the default value altogether?

Reply:
I cannot remember, but I had a valid reason for it at one point (years ago). I'll strip it since it should, in most cases, only be called with a provided arel reference.